x-pack/plugins/security/public/account_management/user_profile/user_profile.tsx

```diff
@@ -513,9 +513,7 @@ export const UserProfile: FunctionComponent<UserProfileProps> = ({ user, data })
           <EuiPageTemplate
             style={{ backgroundColor: euiTheme.colors.emptyShade }}
             className="eui-fullHeight"
-            pageContentBodyProps={{ style: { paddingLeft: 0, paddingRight: 0 } }}
             pageHeader={{
-              style: { paddingLeft: 0, paddingRight: 0 },
```

Comment:
__issue:__ I believe with only these changes we're breaking another design requirement - the title bottom border isn't aligned with the text, see https://github.com/elastic/kibana/pull/132522#pullrequestreview-992067906. We use `bottomBorder` property of the `EuiPageTemplate` that doesn't seem to respect title padding.

I'm not sure what is the recommended way to handle this though, @MichaelMarcialis do you have any suggestion? Should we not use `bottomBorder` property provided by the EUI and "draw" a border with a different approach or is it a bug in EUI that should be fixed (it looks like the user profile page is the only page in Kibana that leverages `bottomBorder` in the page template)?

Reply:
This is an interesting issue. In the EUI documentation examples, the border below the header is nicely aligned with the header and page contents because the default padding is being applied to the containing `.euiPageBody` element. However, in this branch and in EUI's CodeSandbox examples, the border is not aligned to and slightly exceeds the header and page contents because the default padding is being applied individually to `.euiPageHeader` and `.euiPageContentBody`.

@elastic/eui-design, do ya'll know why this disparity is occurring and which is the intended outcome?

**EUI documentation example**

![image](https://user-images.githubusercontent.com/3884767/175363984-e80f5ce2-6ecc-4603-b64e-9a6cb41fb9f7.png)

**This PR's branch**

![image](https://user-images.githubusercontent.com/3884767/175364325-c699fdb1-d254-4b2d-b4e9-c818075abfea.png)

Reply:
It's a limitation that the page template has in combination of border, padding and `restrictWIdth`. The version your pointing to in the docs doesn't have this issue becuase the page header doesn't have `padding` the padding is applied to the whole content wrapper.

My suggestion is to leave all the template defaults (page header is shaded and without the bottom border). There's a new template coming that will solve most of these problems but you're going to fight against this too much just to get the border.

Reply:
Thanks for looking into this @MichaelMarcialis and @cchaos! Yeah, `EuiHorizontalRule` sounds like a viable option for the time being.

Reply:
Heya @cchaos @MichaelMarcialis @azasypkin !

When using the child element `EuiPageHeader` with `bottomBorder={true}`, the border seems to match the content. Does this seem like an acceptable fix?

<img width="1220" alt="Screen Shot 2022-06-27 at 9 36 31 AM" src="https://user-images.githubusercontent.com/21210601/175954799-f6b36955-8ee9-461e-af21-f91d3b6b1fa4.png">

Reply:
@kc13greiner: While this does visually align the border with the left and right edges of the page contents, I don't believe we're supposed to nest an `EuiPageHeader` directly in `EuiPageTemplate` as done here. Doing so ends up putting `.euiPageHeader` as a child of `.euiPageContentBody`, when it is actually supposed to be a sibling in the resulting markup.

If using `EuiPageHeader`, I believe the intent is to no longer use `EuiPageTemplate` and instead nest it in `EuiPage` and `EuiPageBody` components (see [examples in EUI docs](https://elastic.github.io/eui/#/templates/page-template)). That said, I imagine that taking this corrected approach will cause the border to again exceed the width of the page content.

For the sake of simplicity and being able to easily adopt the future changes to the `EuiPageTemplate` that @cchaos mentioned above, let's just go with the components default styles for now (i.e. no bottom border and no background color for header).

Reply:
@MichaelMarcialis Ok! The code is now using the `pageHeader` prop and no bottom border